index.html

```diff
@@ -9,14 +9,24 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <script src="script.js"></script>
 </head>
 <body>
     <div>
         <nav class="hdr">
             <div class="logo">
                 <p>My Logo</p>
             </div>
-            <div class="bar"><img src="./images/Union.png" alt="hamburger menu"></div>
+            <div id="myButtons" class="overlay">
+                <a href="javascript:void(0)" class="closebtn" onclick="closeMenu()">&times;</a>
+                <div class="overlay-content">
+                    <a href="#portfoliostart">Portfolio</a>
+                    <a href="#aboutmelink">About</a>
+                    <a href="#contactmelink">Contact</a>
+                </div>               
+            </div>
+            <span class="hamburger" onclick="openMenu()">&#9776;</span>
```

Comment:
- [ ] OPTIONAL
To improve the **user experience** it will be good practice to ensure that the _hamburger menu_ and the _close button_ are behaving as buttons when the mouse hovers over them. We can improve upon this aspect by adding the CSS attribute `cursor: pointer` to the _hamburger_ and the _close button_ on _mouse hover_ events. We can additionally change the color or opacity of these objects during the hover event.

Reply:
Thank you for the suggestion, applied the attribute!